docs/_posts/2017-10-19-diversity-open-source.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "Diversity in Open Source, and Jekyll's role in it"
+date: 2017-10-19 21:33:00 +0200
+author: pup
+categories: [community]
+---
+
+Open Source has a problem with diversity. GitHub recently conducted a [survey](http://opensourcesurvey.org/2017) that revealed that 95% of the respondents were male, making the demographic even worse than in tech overall. Every other week, I seem to hear of another case of a maintainer engaging in targeted harassment against minorities. People somehow deem it completely okay to let these things slide, though.
```

Comment:
*GitHub recently conducted a survey `which` revealed that 95% of the respondents were male,*
The following clause *"..making the demographic even worse than in tech overall.."* doesn't flow well.. Consider rewording it.

*"Every other week, `I seem to come across a case of a maintainer` engaging in targeted harassment against minorities".* Also, do you want it to be in first-person or will a projected p.o.v be better: *"Every other week, `one` seem to.."*